### DISCUSSION TITLE
ND bug

### DISCUSSION THREAD

**Discussion by ricouw** [flybywiresim/aircraft#4665] [Help] [answered]
Hello,
firstly, well done for your great job on the a320 !!
I regularly update the A320NX (dev) but for a few days the ND (captain side) has been blocked from the start: no information from the start of the aircraft. While the ND (co-pilot side) is working.
In addition, the behavior of the aircraft in flight is very unstable.
Is the problem known?

Thx for your answer

**Comment by ExampleWasTaken** [comment, marked as answer]
Please remove all content from your community folder and make sure you are using the latest version of the A32NX. This will help rule out mod conflicts.

**Reply by ricouw** [reply]
Thanks for your answer but I have already done this but nothing changes...

**Reply by ExampleWasTaken** [reply]
Did you really remove everything. The `flybywire-aircraft-a320-neo` folder must be the only thing inside the community folder? A lot of times people leave things like scenery or liveries in, which can both cause the A32NX to break.

**Reply by ricouw** [reply]
I reinstalled the flybywire-aircraft-a320-neo. It works but it doesn't work with all my other mods. Too bad because there was no problem before.
Can we imagine that the problem will be solved by fbw ?

**Reply by ExampleWasTaken** [reply]
> I reinstalled the flybywire-aircraft-a320-neo. It works but it doesn't work with all my other mods. Too bad because there was no problem before.
> Can we imagine that the problem will be solved by fbw ?

The main focus is not to be compatible with other mods. Try adding them back in one by one. That way you will find the one that is incompatible.

**Reply by ricouw** [reply]
Ok, thank you for your help.
